### ISSUE TITLE
NameError: name 'platform_release' is not defined

### ISSUE THREAD

**Issue by jcandan** [playi/WonderPy#10]
I got the following when trying to `pip install WonderPy`.

```
$ python -V
Python 2.7.9

$ pip -V
pip 1.5.6 from /Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages (python 2.7)

$ pip list
Adafruit-BluefruitLE (0.9.9)
future (0.16.0)
pip (1.5.6)
setuptools (7.0)
wsgiref (0.1.2)

$ pip install WonderPy
Downloading/unpacking WonderPy
  Downloading WonderPy-0.1.0-py2-none-any.whl (497kB): 497kB downloaded
Downloading/unpacking PyObjC (from WonderPy)
  Downloading pyobjc-4.2.2-py2-none-any.whl
Cleaning up...
Exception:
Traceback (most recent call last):
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/basecommand.py", line 122, in main
    status = self.run(options, args)
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/commands/install.py", line 278, in run
    requirement_set.prepare_files(finder, force_root_egg_info=self.bundle, bundle=self.bundle)
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/req.py", line 1265, in prepare_files
    req_to_install.extras):
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/_vendor/pkg_resources.py", line 2291, in requires
    dm = self._dep_map
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/_vendor/pkg_resources.py", line 2484, in _dep_map
    self.__dep_map = self._compute_dependencies()
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/_vendor/pkg_resources.py", line 2517, in _compute_dependencies
    common = frozenset(reqs_for_extra(None))
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/_vendor/pkg_resources.py", line 2514, in reqs_for_extra
    if req.marker_fn(override={'extra':extra}):
  File "/Users/jcandan/.pyenv/versions/2.7.9/lib/python2.7/site-packages/pip/_vendor/_markerlib/markers.py", line 113, in marker_fn
    return eval(compiled_marker, environment)
  File "<environment marker>", line 1, in <module>
NameError: name 'platform_release' is not defined

Storing debug log for failure in /Users/jcandan/.pip/pip.log
```

**Comment by jcandan** [comment]
I think the fix is that (pyobjc)[https://pythonhosted.org/pyobjc/install.html] requires XCode, which I did not yet have on my new Mac. Will report back whether this was the fix or not.

**Comment by jcandan** [comment]
Yep. That fixed it. `xcode-select --install` should work in most cases. 

I ran into trouble and had to actually install XCode from the App Store, then download and install Command Line Tools from the Apple Developer site (https://developer.apple.com/download/more/). There, you can search for Command Line Tools, find the appropriate file based on your version of OS X and XCode, download and install Command Line Tools. 

Once installed, you may need to uninstall/reinstall Python 2.x.

```
$ rm -rf ~/.pyenv/versions/2.7.14
$ pyenv install 2.7.14
$ pip install WonderPy
```

> Note: your setup might be slightly different if you have a different version or are using virtualenv.

**Comment by jcandan** [comment]
Marking closed, and relating to #11
